src/gooseberry/markdown.rs

```diff
@@ -19,6 +19,10 @@ use crate::EMPTY_TAG;
 #[derive(Debug)]
 pub struct MarkdownAnnotation<'a>(pub &'a Annotation);
 
+pub fn replace_spaces(astring: String) -> String {
```

Comment:
This function could take `&str` as input instead of `String` (see the signature of [replace](https://doc.rust-lang.org/std/string/struct.String.html#method.replace)). This makes the function calls cleaner and avoids an extra clone.

Reply:
Yes. Thank you for your advice. I have modified according to this.
It indeed looks nicer.